src/applications/vaos/new-appointment/components/DateTimeSelectPage/index.jsx

```diff
@@ -52,15 +46,24 @@ export function getOptionsByDate(
   }, []);
 }
 
-function ErrorMessage({ facilityId, startRequestFlow }) {
+function startRequestFlow({ requestAppointmentDateChoice, history }) {
+  return requestAppointmentDateChoice(history);
+}
```

Comment:
Similar to my comment above, since `startRequestFlow()` is just a 1 line function and isn't used multiple times, you can just replace the onClick below on line 63.